src/components/MenuProductModal/MenuProductModal.tsx

```diff
@@ -0,0 +1,149 @@
+import PlaceHolderImage from 'assets/burger_figma.png';
+// import { CheckBoxIcon } from 'components/Icons/CheckBoxIcon';
+
+interface Props {
+  product: ProductDto;
+  disableShow: (isShow: boolean) => void;
+}
+
+export const MenuProductModal = ({ product, disableShow }: Props): ReactElement => {
+  const { addToLocalOrder, removeProduct, localOrder } = useLocalOrder();
+  const checkQuantity = (product: ProductDto): number => {
+    const existingItem = localOrder.lineItems.find((item) => item.product.id === product.id);
+    return existingItem ? existingItem.quantity : 1;
+  };
+  const [counter, setCounter] = useState(checkQuantity(product));
+  const increase = (): void => {
+    setCounter((counter) => counter + 1);
+  };
+  const decrease = (): void => {
+    if (counter) {
+      setCounter((counter) => counter - 1);
+    } else {
+      null;
+    }
+  };
+
+  const addToOrder = useCallback(() => {
+    addToLocalOrder(product, counter);
+  }, [addToLocalOrder, product, counter]);
+
+  // const ingredients = ['Patty', 'Bun', 'Lettuce', 'Ketchup', 'Hummus', 'Tahini'];
+  // const addOns = ['Patty', 'Olives', 'Tofu', 'Hummus', 'Tahini'];
+  const allergens = ['Milk', 'Peanuts', 'Celery', 'Soy', 'Gluten'];
+
+  return (
+    <>
+      <div
+        className="fixed inset-0 z-50 flex items-center justify-center overflow-x-auto overflow-y-auto
+       bg-black bg-opacity-25 outline-none focus:outline-none"
+        onClick={() => disableShow(false)}
+      >
+        <div
+          className="min-h-5/6 relative mb-7 flex w-5/12 flex-col items-center justify-between gap-2 rounded-2xl bg-white p-6 shadow-[0_4px_4px_rgba(0,0,0,0.25)]"
+          onClick={(e) => e.stopPropagation()}
+        >
+          <div className="flex flex-row gap-6">
+            <img alt="burger" className="h-[172.41px] min-w-[264px] grow rounded-lg" src={PlaceHolderImage} />
+            <div className="flex flex-col items-start gap-2 p-4">
+              <h5 className="font-poppins text-xl font-semibold text-[#333F4C]">{product.name}</h5>
+              <p className="font-poppins text-left text-[#333F4C]">{product.description}</p>
+              <div className="flex w-full items-center justify-between gap-2">
+                <div className="font-poppins font-semibold text-[#333F4C]">V, Ve, GF</div>
+                <div className="font-poppins flex flex-row items-center justify-between gap-2 text-[#333F4C]">
+                  <button className="flex h-10 w-10 items-center gap-3.5 rounded-md bg-[#E4EAEB] p-4" onClick={decrease}>
+                    -
+                  </button>
+                  <div>{counter}</div>
+                  <button className="flex h-10 w-10 items-center gap-3.5 rounded-md bg-[#E4EAEB] p-4" onClick={increase}>
+                    +
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          {/* Ingredients and AddOns
```

Comment:
remove comments - don't think ingredients and addOns will be implemented in API for a while so best to just get rid for now